### PR TITLE
Easy cpp mixing

### PR DESCRIPTION
**Pull request by BytesGalore**
This provides a little test app for c++ mixing. It relies on https://github.com/RIOT-OS/RIOT/pull/412.

### CODE REVIEW

**Comment by ghost** [comment]
Merged build triggered.

**Comment by ghost** [comment]
Merged build started.

**Comment by ghost** [comment]
Merged build finished.

File: test_cppMix/Makefile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+valgrind: CFLAGS += -g
+valgrind: CFLAGS += -DHAVE_VALGRIND_VALGRIND_H
```
Comment (LudwigKnuepfer) [inline test_cppMix/Makefile:2]: obsolete

Comment (LudwigKnuepfer) [inline test_cppMix/Makefile:2]: I think it's best to rebase the whole file based on the current dist/Makefile...

Comment (BytesGalore) [inline test_cppMix/Makefile:2]: true. this test is outdated and superseded by #770

Comment (BytesGalore) [inline test_cppMix/Makefile:2]: https://github.com/RIOT-OS/RIOT/pull/770

Comment (LudwigKnuepfer) [inline test_cppMix/Makefile:2]: Care to close it then?

**Comment by BytesGalore** [comment]
moin, 
I close it (this time intended)

**Pull request closed without merging**